hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java

```diff
@@ -151,15 +159,12 @@ public BackgroundTaskResult call() throws Exception {
           List<Pair<String, String>> prefixPairs =
               constructPrefixPairs(snapshotInfo);
 
-          String dbName = OM_DB_NAME + snapshotInfo.getCheckpointDirName();
-
-          String snapshotCheckpointDir = omMetadataDir + OM_KEY_PREFIX +
-              OM_SNAPSHOT_CHECKPOINT_DIR;
-          try (RDBStore rdbStore = (RDBStore) OmMetadataManagerImpl
-              .loadDB(ozoneManager.getConfiguration(),
-                      new File(snapshotCheckpointDir),
-                      dbName, true, Optional.of(Boolean.TRUE),
-                      Optional.empty(), false, false)) {
+          try (ReferenceCounted<IOmMetadataReader, SnapshotCache>
+                   snapshotMetadataReader = snapshotCache.get()
+              .get(snapshotInfo.getTableKey())) {
+            OmSnapshot omSnapshot = (OmSnapshot) snapshotMetadataReader.get();
+            RDBStore rdbStore = (RDBStore) omSnapshot.getMetadataManager()
+                .getStore();
             RocksDatabase db = rdbStore.getDb();
```

Comment:
nit: This function is marked visible for testing. Please remove the annotation.
https://github.com/apache/ozone/blob/00acf36d32ea8f452437d2333034ed15d20813ea/hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java#L445

Reply:
done